libadikted/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+##
+##
+##
+
+
+cmake_minimum_required(VERSION 3.8)
+
+
+if( "${CMAKE_CURRENT_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_BINARY_DIR}" )
+    message( FATAL_ERROR "Building in source is forbidden. Change output directory.")
+endif()
+
+
+project( libadikted )
+
+
+if( NOT CMAKE_BUILD_TYPE )
+    ##set( CMAKE_BUILD_TYPE "Debug")
+    set( CMAKE_BUILD_TYPE Debug CACHE STRING "Choose the type of build." FORCE)
+endif()
+
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+    set( CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install" CACHE PATH "Install path prefix" FORCE )
+endif()
+
+
+set(CMAKE_DEBUG_POSTFIX _d)
+
+
+##
+## set default compilation standard
+##
+if (CMAKE_VERSION VERSION_LESS "3.1")
```

Comment:
Is there any point to this if statement when minimum version is already specified to version 3.8 on line 6?

Reply:
In this particular case there isn't. Cleaned up.